Student/Student/CanvasAppDelegate.swift

```diff
@@ -111,14 +111,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate, AppEnvironmentDelegate {
         } })
     }
 
-    func application(_ application: UIApplication, handleOpen url: URL) -> Bool {
-        return openCanvasURL(url)
-    }
-
-    func application(_ application: UIApplication, open url: URL, sourceApplication: String?, annotation: Any) -> Bool {
-        return self.application(application, handleOpen: url)
-    }
-
```

Comment:
these are deprecated in favor of the one below. Only Student seems to implement them in addition to `open url`. Is it possible implementing these causes the `Copy to Student` bug?